worker/src/features/batchExport/handleBatchExportJob.ts

```diff
@@ -333,8 +327,8 @@ export const handleBatchExportJob = async (
   const endpoint = env.S3_ENDPOINT;
   const region = env.S3_REGION;
 
-  if (!accessKeyId || !secretAccessKey || !bucketName || !endpoint || !region) {
-    throw new Error("S3 credentials not found");
+  if (!bucketName) {
+    throw new Error("No S3 bucket configured for exports.");
```

Comment:
This is the only non-prettier/linting change in this file. Again, we only validate that there is a bucket and rely on S3 errors if further credentials are missing.